api/events.js

```diff
@@ -40,3 +36,15 @@ export const findByIds = (apiUrl, headers, fetch) => async (eventIds) => {
   const events = await Promise.all(mappedEvents);
   return events.filter((event) => !event.status);
 };
+
+export const getCalendarSummary = (apiUrl, headers, fetch) => async ({
+  id,
+  locale,
```

Comment:
if you want to make sure it uses the defaults when you don't pass anything you need to add ` = {}` after the destructuring